doc/whats_new.rst

```diff
@@ -85,6 +85,8 @@ API
 
     - Deprecated the `baseline` parameter in :class:`mne.Evoked`, by `Felix Raimundo`_
 
+    - The API of :meth:`mne.SourceEstimate.plot` and :func:`mne.viz.plot_source_estimates` has been updated to reflect current PySurfer 0.6 API. The ``config_opts`` parameter is now deprecated and will be removed in mne 0.14, and the default representation for time will change from ``ms`` to ``s`` in mne 0.14. By `Christian Brodbeck`_
```

Comment:
Agreement attraction? :)


> On Jul 7, 2016, at 12:17 PM, Eric Larson <notifications@github.com> wrote:
> 
> In doc/whats_new.rst <https://github.com/mne-tools/mne-python/pull/3391#discussion_r69937829>:
> 
> > @@ -85,6 +85,8 @@ API
> >  
> >      - Deprecated the `baseline` parameter in :class:`mne.Evoked`, by `Felix Raimundo`_
> >  
> > +    - The API of :meth:`mne.SourceEstimate.plot` and :func:`mne.viz.plot_source_estimates` has been updated to reflect current PySurfer 0.6 API. The ``config_opts`` parameter is now deprecated and will be removed in mne 0.14, and the default representation for time will change from ``ms`` to ``s`` in mne 0.14. By `Christian Brodbeck`_
> ahh no I misread it
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/mne-tools/mne-python/pull/3391/files/7eef2cb902ce4e1c0abd973d53a45dbd21529c6a#r69937829>, or mute the thread <https://github.com/notifications/unsubscribe/AAI5a-c8Ri6Dks8xhMqkpHlNHqo3Md4oks5qTSafgaJpZM4JFteX>.
>